testdata/source/schwab_csv/test_basic/import_results.beancount

```diff
@@ -582,6 +582,28 @@
 
 2020-11-15 price SMAL040120203200P                   12.53 USD
 
+;; date: 2020-11-19
+;; info: {"filename": "<testdata>/test_basic/transactions/Brokerage_Transactions_20201115-180021.CSV", "line": 3, "type": "text/csv"}
+
+; features: []
+2020-11-19 * "INTEREST - PROMOTIONAL AWARD"
```

Comment:
Nit comment, but can you add an if in the BankInterest class so the narration prefix for promo award is not INTEREST, but something else. Probably INCOME?

Reply:
Done.